tests/collections/multimodal/test_speechllm_models.py

```diff
@@ -218,3 +218,24 @@ def test_predict_step(self, llm_model_config, perception_model_config, trainer_c
         response = model.predict_step(batch, 0, 0)
         ground_truth = 'to suit you. Please note these are lecture notes from an alternate presentation. Copyright  ⁇ '
         assert response['sentences'][0] == ground_truth
+
```

Comment:
could you also confirm the changes in inference do not break test_validation_step and test_predict_step
also rerun some zero-shot inference to confirm the changes won't affect results there